rust/src/builtins/apply_live.rs

```diff
@@ -91,12 +92,21 @@ pub(crate) fn applylive_finish(sysroot: &crate::ffi::OstreeSysroot) -> CxxResult
     };
     pkgdiff.print();
 
-    if pkgdiff.n_removed() == 0 && pkgdiff.n_modified() == 0 {
-        crate::ffi::output_message("Successfully updated running filesystem tree.");
-    } else {
-        crate::ffi::output_message(
-            "Successfully updated running filesystem tree; some services may need to be restarted.",
-        );
+    if pkgdiff.n_removed() != 0 || pkgdiff.n_modified() != 0 {
+        let diff = ostree_ext::diff::diff(
+            repo,
+            &booted_commit,
+            &live_state.commit.as_str(),
+            Some("/usr/lib/systemd/system"),
+        )?;
+        if !diff.changed_files.is_empty() {
+            let output = Command::new("/usr/bin/systemctl")
+                .arg("daemon-reload")
+                .output()
```

Comment:
Use `.status()` here - `.output()` is for piping the stdout of the process, but we don't need that.